app/src/services/system-checks/DeviceInvalidatedSystemCheck.ts

```diff
@@ -41,8 +42,8 @@ export class DeviceInvalidatedSystemCheck implements SystemCheckStrategy {
     try {
       const idToken = await this.getIdToken()
 
-      if (idToken.bcsc_event === BCSCEvent.Cancel && idToken.bcsc_reason === BCSCReason.CanceledByAgent) {
-        // Device has been invalidated by the agent, user must re-authorize the device
+      if (idToken.bcsc_event === BCSCEvent.Cancel) {
+        this.caseType = idToken.bcsc_reason
         return false
       }
```

Comment:
question: This logical change is less strict than before, are we sure this is what we wan't now?